### ISSUE TITLE
Support request: folder contains shared content

### ISSUE THREAD

**Issue by clovergaze** [astrada/google-drive-ocamlfuse#699]
I set `root_folder` to a specific folder in my Google Drive, but for some reason the folder (locally) also contains all the shared content (`.shared`) and all the trash (`.Trash`).

I managed to remove the trash with `disable_trash=true`, but the `shared_with_me=false` option gets removed from the config file and I can not find the new option to disable the shared content.

I want the local folder to just contain the content of the remote root folder and its sub-folders, nothing else.

How can that be achieved?

**Comment by astrada** [comment]
No, sorry, there is no way to omit the `.shared` folder because otherwise if you have Shortcuts in your Drive, they would be dangling symlinks.

**Comment by clovergaze** [comment]
@astrada I understand. But maybe you could add this option anyway for mounted folders that do not have shortcuts to shared content, so the user could configure it at his own discretion?

**Comment by lukasz18dg** [comment]
That would be great, in my case: my local Google Drive folder suddenly increased from 20GB to 40GB, because someone has something big to share with me, where I don't need it on local disk.
